README.md

```diff
@@ -294,6 +294,60 @@ offset in a string. `TEXT` must be contained at the location specified.
 
 ---
 
+## Presence
+
+`json0` has some limited support for presence information: information about
+clients' transient position within a document (eg their cursor or selection).
+
+It also supports presence in `text0`.
+
+### Format
+
+#### `json0`
+
+The format of a `json0` presence object follows a similar syntax to its ops:
+
+    [{p: ['key', 123], v: 0}]
```

Comment:
From the implementation, this shouldn't have an array wrapping it?

Reply:
Yup, docs were wrong. I've updated.